packages/dds/merge-tree/api-report/merge-tree.api.md

```diff
@@ -249,9 +249,15 @@ export interface Dictionary<TKey, TData> {
     remove(key: TKey): void;
 }
 
+// @internal (undocumented)
+export function disableStrictPartialLengthChecks(): void;
```

Comment:
can we export these from the test index only?

Reply:
Oh, can other packages now reach into `merge-tree/test`?

Reply:
Yeah, see package.json exports:

![image](https://github.com/microsoft/FluidFramework/assets/5222607/122bbf03-61c6-441e-88e3-5949fb033932)

so `import { disableStrictPartialLengthChecks } from "@fluidframework/merge-tree/dist/test"` should work.

we do this a few places in sequence and I think e2e tests to reuse some test utilities

Reply:
might be nice to just expose this as a function which invokes both mocha hooks then, that should simplify some test code and help make sure we don't accidentally leave it on for a test suite.

Reply:
Oh, I didn't know you could programmatically invoke mocha hooks. If you call beforeEach twice, will both run or will one override the other?

Reply:
They'll both be run. Mocha runs all hooks for each type in the order they're defined, so outermost suite aka global hooks -> innermost. I get the impression the implementation just adds it to a list on the test run state object. Never bothered to look though. See [here](https://mochajs.org/#hooks).

Reply:
You can do some neat stuff with that structure btw, see e.g. https://github.com/microsoft/FluidFramework/pull/18919/files